docs/getting-started/part-2/delta-lake-format.md

```diff
@@ -113,7 +113,7 @@ But state-of-the-art is to use notebooks like Jupyter for this.
 One of the most advanced notebooks for Scala code we found is Polynote, see [polynote.org](https://polynote.org/).
 
 We will now start Polynote in a docker container, and an external Metastore (Derby database) in another container to share the catalog between our experiments and the notebook.
-To do so, run the following commands in the projects root directory:
+To do so you need to add additional files to the project. Change to the projects root directory and **unzip part2.additional-files.zip**, then run the following commands in the projects root directory:
```

Comment:
@zzeekk :
https://github.com/smart-data-lake/getting-started/blob/master/part2.additional-files.zip
contains the solution of part2. When you unzip it, you might think you need to replace your application.conf file with it's contents and then you might be confused, which happened to me. OK if I remove it from the zip in PR  https://github.com/smart-data-lake/getting-started/pull/2 ?

Reply:
Moreover: @pgruetter can you check if unzipping this on linux will work without any newline-issues?

Reply:
Good point. I don't have it set up to quickly test all files. But in Linux, vi at least shows that it's a Windows file, so we might have a problem with them, yes:
![image](https://user-images.githubusercontent.com/6121700/151112214-47a96dc9-7224-42ff-ad5f-08d5f6fd1357.png)

And you're right, this also got me confused:
![image](https://user-images.githubusercontent.com/6121700/151112373-35fa4241-3f98-45a8-aaf5-7c708d423817.png)

Reply:
Yes, we should remove the application.conf from part2.additional-files.zip.

For the line-endings: this is because i edited and zipped it in Windows. Git probably doesn't convert line-endings inside zip-files when pushing. It's good if you can test if it works on Linux with wrong line-endings!